packages/storybook/.storybook/main.js

```diff
@@ -3,21 +3,21 @@ module.exports = {
   typescript: {
     check: true,
     checkOptions: {},
-    reactDocgen: 'react-docgen-typescript',
-    reactDocgenTypescriptOptions: {
-      shouldExtractLiteralValuesFromEnum: true,
-      propFilter: prop => {
-        if (/node_modules/.test(prop && prop.parent && prop.parent.fileName)) {
-          return false
-        }
-        return true
-      },
-    },
+    reactDocgen: false,
```

Comment:
It seems to be working for me locally if we re-enable this and the addon below.  Could we bring it back?  If it was failing it's possible there's a build caching issue with `packages/storybook/node_modules/.cache/`

```suggestion
    reactDocgen: true,
```